LICENSE

```diff
@@ -1,13 +1,201 @@
-Copyright 2023 Core Mesa Team and contributors
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
 
-    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Also, today is 2024, so this needs to be `https://`, not `http://`, see https://www.apache.org/foundation/license-faq.html#Apply-My-Software.